packages/shared/client/ReplayClient.ts

```diff
@@ -371,6 +371,14 @@ export class ReplayClient implements ReplayClientInterface {
     return countsObject;
   }
 
+  async getAllEventHandlerCounts(): Promise<Record<string, number>> {
+    const sessionId = this.getSessionIdThrows();
+    const { counts }: { counts: { type: string; count: number }[] } =
+      await client.Debugger.getAllEventHandlerCounts({}, sessionId);
```

Comment:
`Debugger.getAllEventHandlerCounts` is not a function in the protocol package version installed in devtools, and it looks like we're depending on the latest published version of that package.

I think you need to publish an updated version of `@replayio/protocol` to NPM, then update `package.json` and `yarn.lock` in this repo to include that version as part of this PR.

Reply:
Yep. Working on that.
It is still in a `BAC` PR (https://github.com/replayio/backend/pull/6951), which is also failing checks /o\

Reply:
Going to hold off on reviewing this until I can build and run it locally. Let me know when that's ready!